action.yml

```diff
@@ -8,11 +8,9 @@ inputs:
   unset-previous:
     description: Whether to unset environment variables populated by 1Password in earlier job steps
     default: false
+  export-env:
+    description: Export the secrets as environment variables
+    default: false
 runs:
-  using: composite
-  steps:
-    - shell: bash
-      env:
-        INPUT_UNSET_PREVIOUS: ${{ inputs.unset-previous }}
-      run: |
-        ${{ github.action_path }}/entrypoint.sh
+  using: 'node12'
```

Comment:
Change to `node16`, since that is the most recent one.

Reply:
changed